app-store-vsts-extension.json

```diff
@@ -46,8 +46,15 @@
         },
         "support": {
             "uri": "https://github.com/microsoft/app-store-vsts-extension/issues"
+        },
+        "issues": {
+            "uri": "https://github.com/microsoft/app-store-vsts-extension/issues"    
         }
     },
+    "repository": {
```

Comment:
Going as per docs here which are not very clear: https://docs.microsoft.com/en-us/vsts/extend/develop/manifest#CustomerQnASupport